changelog.md

```diff
@@ -22,7 +22,7 @@
 
 ## v0.3.4 2017-05-01
 
-* performance improvement of HashDiff#similar? #31
+* performance improvement of Hashdiff#similar? #31
```

Comment:
I wouldn't change these 2 changelog messages, as your message from 0.4 onwards will be changing of the global scope.

Reply:
I was thinking about it and I'm not sure about this change. @liufengyun what do you think?

Reply:
Two options are ....
1) Revert change
2) (Preferred), use the syntax for `.` and `#` exclusively. So here this would say 

```
* performance improvement of `#similar?` #31
```

And the one below would say ...

```
add more specs for `.best_diff`
```

Reply:
@luke-hill you're totally right.  Thanks for the catch!

Reply:
Amended the commit.